src/jace/api.py

```diff
@@ -22,44 +22,47 @@
 __all__ = ["JITOptions", "grad", "jacfwd", "jacrev", "jit"]
 
 _P = ParamSpec("_P")
-_R = TypeVar("_R")
 
 
 class JITOptions(TypedDict, total=False):
     """
     All known options to `jace.jit` that influence tracing.
 
-    Note:
-        Currently there are no known options, but essentially it is a subset of some
-        of the options that are supported by `jax.jit` together with some additional
-        JaCe specific ones.
+    Not all arguments that are supported by `jax-jit()` are also supported by
+    `jace.jit`. Furthermore, some additional ones might be supported.
+    The following arguments are supported:
+    - `backend`: For which platform DaCe should generate code for. It is a string,
+        where the following values are supported: `'cpu'` or `'gpu'`.
+        DaCe's `DeviceType` enum or FPGA are not supported.
     """
 
+    backend: str
```

Comment:
```suggestion
    backend: Literal['cpu', 'gpu']
```

Reply:
I do not think that enforcing this through the annotation is the right way.

Reply:
Why not? What's the use case for the `Literal` type annotation then? Using `Literal` here also helps in  documenting the supported values in a more prominent way.

Reply:
First a practical reason, if you read the backend string say from a file then you will get a `str`, so you will have to cast it.
It also does not really helps documenting since in _general_ it does not explain what a particular value mean or does, you still have to read the doc anyway.